autoload/javacomplete/classpath/gradle.vim

```diff
@@ -73,9 +73,9 @@ function! s:GenerateClassPath(path, gradle) abort
   if exists("g:JavaComplete_GradleExecutable")
     let gradle = g:JavaComplete_GradleExecutable
   else
-    let gradle = 'gradle'
+    let gradle = fnamemodify(g:JavaComplete_GradlePath, ':p:h') . (javacomplete#util#IsWindows() ? '\gradlew.bat' : '/gradlew')
```

Comment:
yes,it should be  absolute path ,because if you do not use vim-rooter,maybe your current dir is not the root of the project.